docs/streams/developer-guide.html

```diff
@@ -2827,22 +2834,20 @@
 
     <p>
         For changelog topics you can also override the default configs on a per store basis.
-        This can be done by using any method overload that has a <code>StateStoreSupplier</code> as a parameter:
+        This can be done by using any method overload that has a <code>Materialized</code> as a parameter:
     </p>
 
     <pre class="brush: java;">
         // a map to add topic config
         Map&lt;String, String&gt; topicConfig = new HashMap<>();
```

Comment:
Not introduced in this PR, but do we need `&lt;&gt;` instead of `<>` here?

Reply:
Yeah, I think so, too.